docs/sorting-algorithms/insertion-sort.mdx

```diff
@@ -0,0 +1,194 @@
+The above steps are repeated until all the elements in the list are in their 
+appropriate position.i.e. list is in sorted order.*/
+
+
+
+
+#include <iostream>
+using namespace std;
+
+//Function to print array.
+void display(int arr[], int size) 
+{ 
+    int i; 
+    for (i=0; i < size; i++) 
+    cout<< arr[i]<<"\t"; 
+    cout<<"\n"; 
+} 
+
+//Main function to run the program.
+int main() 
+{ 
+    int array[] = {5, 3, 1, 9, 8, 2, 4,7}; 
+    int size = sizeof(array)/sizeof(array[0]); 
+
+    cout<<"Before Insertion sort: \n";
+    display(array, size);//Using dispaly function to print unsorted array.
+
+    int i, target, j;  
+    for (i = 1; i < size; i++) 
+    {  
+        target = array[i];  
+        j = i - 1;  
+
+        /* Here the elements in b/w arrary[0 to i-1]
+        which are greater than target are moved
+        ahead by 1 position each*/
+        while (j >= 0 && array[j] > target) 
+        {  
+            array[j + 1] = array[j];  
+            j = j - 1;  
+        }  
+        array[j + 1] = target;  
+    }
+    cout<<"After Insertion sort: \n"; 
+    display(array, size);//Using dispaly function to print sorted array. 
+    return 0; 
+}
+```
+
+</TabItem>
+
+<TabItem value="py">
+
+```python
+"""
+This is a pure Python implementation of the insertion sort algorithm
+"""
+
+""" Implementation of the insertion sort algorithm in Python
+    :parameter array: A mutable ordered collection with comparable items inside
+    
+    :return : the same collection ordered by ascending
+    Examples:
+    >>> insertion_sort([0, 5, 3, 2, 2])
+    [0, 2, 2, 3, 5]
+    >>> insertion_sort([])
+    []
+    >>> insertion_sort([-2, -5, -45])
+    [-45, -5, -2]
+"""
+def insertion_sort(array):
+    # Loop from the second element of the array until
+    # the last element
+    for i in range(1, len(array)):
+        # This is the element we want to position in its
+        # correct place
+        key_item = array[i]
+
+        # Initialize the variable that will be used to
+        # find the correct position of the element referenced
+        # by `key_item`
+        j = i - 1
+
+        # Run through the list of items (the left
+        # portion of the array) and find the correct position
+        # of the element referenced by `key_item`. Do this only
+        # if `key_item` is smaller than its adjacent values.
+        while j >= 0 and array[j] > key_item:
+            # Shift the value one position to the left
+            # and reposition j to point to the next element
+            # (from right to left)
+            array[j + 1] = array[j]
+            j -= 1
+
+        # When you finish shifting the elements, you can position
+        # `key_item` in its correct location
+        array[j + 1] = key_item
+
+    return array
+
+
+
+if __name__ == "__main__":
+
+
+    user_input = list(map(int, input("Enter a sequence of comma seprated numbers:\n").split(',')))
```

Comment:
Can we use the same console statements for both C++ and Python. Let's move towards a bit of standardization